### PR TITLE
#603 judgment content files webapp resource handler

### PR DESCRIPTION
**Pull request by madryk**

### CODE REVIEW

File: saos-webapp/src/main/java/pl/edu/icm/saos/webapp/WebappConfiguration.java [hunk 1]
```diff
@@ -67,6 +70,8 @@ public TilesConfigurer tilesConfigurer() {
     public void addResourceHandlers(ResourceHandlerRegistry registry) {
         registry.addResourceHandler("/static/**").addResourceLocations("/WEB-INF/static/").setCachePeriod(3600*24*7);
         registry.addResourceHandler("/robots.txt").addResourceLocations("/WEB-INF/").setCachePeriod(0);
+        registry.addResourceHandler("/files/judgments/**").addResourceLocations(ResourceUtils.FILE_URL_PREFIX + judgmentsContentPath).setCachePeriod(3600*24);
```
Comment (lukdumi) [inline saos-webapp/src/main/java/pl/edu/icm/saos/webapp/WebappConfiguration.java]: I would give much much smaller cache (for example 10minutes). Consecutive reading of the same file will be rare and will probably be done in short time period.

File: saos-webapp/src/main/resources/saos.default.properties [hunk 2]
```diff
@@ -56,6 +56,9 @@ import.judgments.nationalAppealChamber.content.dir
 # path to directory with downloaded national appeal chamber judgments import content files
 import.judgments.nationalAppealChamber.download.dir
 
+# path to base directory where judgment content files are stored
+judgments.content.dir
```
Comment (lukdumi) [inline saos-webapp/src/main/resources/saos.default.properties]: Shouldn't we have a default dir set here?

File: saos-webapp/src/main/java/pl/edu/icm/saos/webapp/WebappConfiguration.java (hunk 1 above)
Comment (madryk) [inline saos-webapp/src/main/java/pl/edu/icm/saos/webapp/WebappConfiguration.java]: :+1:

File: saos-webapp/src/main/resources/saos.default.properties (hunk 2 above)
Comment (madryk) [inline saos-webapp/src/main/resources/saos.default.properties]: :+1:

**Pull request closed without merging**